build.gradle.kts

```diff
@@ -0,0 +1,132 @@
+import org.jetbrains.intellij.tasks.PatchPluginXmlTask
+import org.jetbrains.intellij.tasks.PublishPluginTask
+import org.jetbrains.intellij.tasks.RunIdeTask
+import org.jetbrains.intellij.tasks.RunPluginVerifierTask
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import java.util.EnumSet
+
+plugins {
+  id("idea")
+  id("java")
+  kotlin("kapt") version "1.8.10"
+  id("org.jetbrains.intellij") version "1.13.3"
+  id("org.jetbrains.kotlin.jvm") version "1.8.10"
+  id("org.jetbrains.kotlin.plugin.serialization") version "1.4.32"
+}
+
+java {
+  sourceCompatibility = JavaVersion.VERSION_17
+  targetCompatibility = JavaVersion.VERSION_11
+}
+
+// The latest supported versions. Note, these are updated automatically from update-major-versions.sh
+val IIC_RELEASE_VERSION = "231.9011.34"
+val IIC_EAP_VERSION = "232.6095.10"
+val GO_RELEASE_VERSION = "231.9011.34"
+val GO_EAP_VERSION = "232.6095.10"
+// The oldest supported versions.
+val IIC_FROM_VERSION = "222.4554.10"
+val GO_FROM_VERSION = "222.4554.12"
```

Comment:
Let's extract these into a `versions.properties` or into `settings.gradle.kts` 😰

Reply:
I've tried it now. It looks like the right way but I don't really like the extra indirection with loading properties by a string key...

Reply:
I've changed this now.